mnefun/_mnefun.py

```diff
@@ -239,6 +240,8 @@ class Params(Frozen):
     sss_origin : array-like, shape (3,) | str
         Origin of internal and external multipolar moment space in meters.
         Default is center of sphere fit to digitized head points.
+    plot_hp : bool
+        Plot head movement traces from raw data
```

Comment:
this docstring is out of date. It's not part of the constructor anyway so let's get rid of it